Mathlib/CategoryTheory/Limits/Lattice.lean

```diff
@@ -14,18 +14,13 @@ import Mathlib.CategoryTheory.Limits.Shapes.FiniteLimits
 -/
 
 
-universe w u
-
-open CategoryTheory
-
-open CategoryTheory.Limits
+universe w w' u
 
 namespace CategoryTheory.Limits.CompleteLattice
 
 section Semilattice
 
-variable {α : Type u}
-variable {J : Type w} [SmallCategory J] [FinCategory J]
+variable {α : Type u} {J : Type w} [Category J] [FinCategory J]
```

Comment:
```suggestion
variable {α : Type u} {J : Type w} [Category.{w'} J] [FinCategory J]
```
To make sure the generalisation is actually happening here!

Reply:
I tried this but `FinCategory` assumes that universes are the same.

Reply:
Ah okay, so there isn't universe generalisation happening in this part, I see. Thanks.